File: Level Two/taskFive.py
number = int(input("Enter number: "))
number_two = int(input('Enter second number: '))
number_three = int(input('Enter third number: '))

if(number > number_two and number > number_three):
    print(number)
if(number_two > number_three and number_two > number):
    print(number_two)
elif(number_three > number_two and number_three > number):
    print(number_three)

File: Level Two/TaskFive.java
import java.util.Scanner;
public class TaskFive{
    public static void main(String[] args){
Scanner inputCollector = new Scanner(System.in);

System.out.print("Enter number: ");
 int number = inputCollector.nextInt();
System.out.print("Enter second number: ");
int secondNumber = inputCollector.nextInt();
System.out.print("Enter third number: ");
int thirdNumber = inputCollector.nextInt();
int largest = number;
if(secondNumber > number && secondNumber > thirdNumber){
System.out.println(secondNumber + " is bigger");
}else if(thirdNumber > secondNumber && thirdNumber > number){
System.out.println(thirdNumber + " is bigger");
}else{
System.out.println(largest);
}




}



}
